tracer.go

```diff
@@ -27,11 +27,12 @@ type Tracer interface {
 	StartSpan(operationName string) Span
 	StartSpanWithOptions(opts StartSpanOptions) Span
 
-	// Return an Injector for the given `format` value, or nil if the Tracer
-	// does not support such a format.
+	// Inject() takes the `toInject` Span instance and represents it for
```

Comment:
I find `toInject` confusing because it sounds like something is to be injected into the span. Just `sp` is better imo.